### PR TITLE
Prepare for v2.3.1 release

### PR DESCRIPTION
**Pull request by koooosh**
**Issue number:** N/a

**Description of changes:**

This includes the following changes:
- `CHANGELOG` update for upcoming 2.3.1 release
- Bumping `release-version` to `2.3.1` in Twoliter
- Creating advisory directory for 2.3.1 with relevant files

**Testing done:** N/a

**Terms of contribution:**

By submitting this pull request, I agree that this contribution is dual-licensed under the terms of both the Apache License, version 2.0, and the MIT license.

### CODE REVIEW

**Comment by koooosh** [comment]
Force push rebases on top of https://github.com/bottlerocket-os/bottlerocket-core-kit/pull/60

**Review by vigh-m — APPROVED** [review]

**Review by yeazelm — APPROVED** [review]

**Review by KCSesh — APPROVED** [review]

**Review by larvacea — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,25 @@
...
+
+* Update bottlerocket-settings-models to v0.2.0 ([#58])
+* Update bottlerocket-sdk to v0.43.0 ([#60])
+
+## Orchestrator Changes
+
+### Kubernetes
+
+* Support hostname-override-source ([#59])
+
```
Comment (monirul) [inline CHANGELOG.md:13]:
```suggestion
* nvidia-container-runtime, nvidia-k8s-device-plugin: support Nvidia settings APIs [#48]
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,25 @@
...
+[#55]: https://github.com/bottlerocket-os/bottlerocket-core-kit/pull/55
+[#58]: https://github.com/bottlerocket-os/bottlerocket-core-kit/pull/58
+[#59]: https://github.com/bottlerocket-os/bottlerocket-core-kit/pull/59
+[#60]: https://github.com/bottlerocket-os/bottlerocket-core-kit/pull/60
+
```
Comment (monirul) [inline CHANGELOG.md:24]:
```suggestion
[#48]: https://github.com/bottlerocket-os/bottlerocket-core-kit/pull/48
```

**Review by monirul — APPROVED** [review]
LGTM. Added a suggestion.

File: CHANGELOG.md (hunk 2 above)
Comment (koooosh) [reply CHANGELOG.md:24]: Ack, will commit this on my own just to preserve number ordering (before 58)

**Comment by koooosh** [comment]
Force push takes above suggestions to add Nvidia change to CHANGELOG

**Review by vigh-m — APPROVED** [review]

**Review by yeazelm — APPROVED** [review]

**Pull request merged**